04-django/random_pswd.py

```diff
@@ -0,0 +1,13 @@
+import random
+import string
+
+
+password = '  '.join(random.choice(string.printable) for i in range(12))
```

Comment:
You could also use `random.choices(string.printiable, k=12)` to directly get 12 random elements.